File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    answer = ''
    se = set()
    remove = set()
    for c in s:
        if c in se:
            remove.add(c)
        else:
            se.add(c)
    se = se - remove
    answer = "".join(sorted(list(se)))
    return answer

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string s) {
    string answer = "";
    string tmp = "";
    for(auto c:s){
        if(count(s.begin(), s.end(),c)==1){
            answer+=c;
        }
    }
    sort(answer.begin(), answer.end());
    return answer;
}